src/main/java/de/fearnixx/jeak/service/token/TokenConfiguration.java

```diff
@@ -81,4 +81,8 @@ public void saveToken(String token, TokenScope tokenScope) {
         child.appendValue(tokenScope.getScopeSet());
         getConfig().put(token, child);
     }
+
+    public boolean deleteToken(String token) {
+        return getConfig().remove(token) != null;
```

Comment:
Should we return ``false`` for nonexistent tokens? Because that would be when ``#remove`` returns ``null``.

Reply:
Hm, that is a good question. I think it makes sense to return false in case of the deletion not working, and I would consider not entering a valid token as not working.

Reply:
While that might be true, I am more a fan of whitebox-behavior, where a #remove call with an nonexistent token would yield true because: The intendet outcome is that the token no longer works - if it didn't work in the first place, I would say that is fine for the invoker.

Reply:
That sounds reasonable enough, I will change it. And to make sure its understandable, I am gonna add that to the javadoc as well.